src/meta/processors/admin/Balancer.cpp

```diff
@@ -313,6 +325,248 @@ StatusOr<HostAddr> Balancer::hostWithMinimalParts(
     return Status::Error("No host is suitable for %d", partId);
 }
 
+cpp2::ErrorCode Balancer::leaderBalance() {
+    folly::Promise<Status> promise;
```

Comment:
Add check if we are balancing data,   reject the leader balance request

Reply:
Good suggestion.